packages/backend/src/models/UserModel.ts

```diff
@@ -541,6 +579,12 @@ export class UserModel {
         return this.getUserDetailsByUuid(user.user_uuid);
     }
 
+    /**
+     * Returns the user with the default organization
+     * Used in old deserialization method, for backwards compatibility
+     * You should use findSessionUserAndOrgByUuid instead
+     * @deprecated
```

Comment:
We could remove this instead of deprecating it by doing a `delete from sessions` migration

Reply:
mm, this is also used in other methods like csv downloads or the new github oauth authentication to get the organizationUuid... 

![image](https://github.com/lightdash/lightdash/assets/1983672/e985ca47-c2c0-4f9c-97bf-73b9904f5ac9)

we should do this on a separate PR, to remove all dependencies to `user.organization`

Reply:
I made the migration and updated the deserializer method, I kept the deprectated method and updated its description